notebooks/pregnancy/maternal_brain_project/r2_pvalues_all_enigma_meshes_new.ipynb

```diff
@@ -0,0 +1,1445 @@
+{
```

Comment:
<p>NIT: I only understood feature-wise r2 means by seeing that 2502*3=7506. I wonder if there could be a different name?</p><p>super clear otherwise.</p><p> </p><p>R2s are not that great. Is it for the pregnancy-only period?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geometric-intelligence/herbrain/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>that evaluator is generic and does not care if we're speaking about features or coordinates of points. I'll make it in way we can rename the "featurewise".</p><p> </p><p>all the weeks for which <code>stage != "post"</code>  </p>